core/domain/exp_jobs_one_off_test.py

```diff
@@ -461,8 +461,8 @@ def test_no_action_is_performed_for_deleted_exploration(self):
             self, exp_jobs_one_off.ExpSummariesContributorsOneOffJob,
             check_error=True,
             error_type=base_models.BaseModel.EntityNotFoundError,
-            error_msg='Entity for class ExpSummaryModel with id 100 not found',
-            function_to_be_called=exp_services.get_exploration_summary_by_id,
+            error_msg='Entity for class ExplorationModel with id 100 not found',
+            function_to_be_called=exp_services.get_exploration_by_id,
```

Comment:
I'm not sure about this change as the job name is `ExpSummariesContributorsOneOffJob` related to exp summary why we are changing this? (Ditto below)

Reply:
Well now get_exploration_summary_by_id returns None with an invalid exp id but you are right we should be using get_exploration_summary_by_id not get_exploration_by_id. In this case, we would not be getting an error. I modified the construct according to this change